sentry-android-core/src/test/java/io/sentry/android/core/PerformanceAndroidEventProcessorTest.kt

```diff
@@ -0,0 +1,100 @@
+package io.sentry.android.core
+
+import com.nhaarman.mockitokotlin2.mock
+import io.sentry.SentryTracer
+import io.sentry.protocol.SentryTransaction
+import java.util.Date
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class PerformanceAndroidEventProcessorTest {
```

Comment:
`m`: Nice tests. It would be nice to have some async tests to make sure the measurement is only sent once and to validate if the measurement value is correct. As pointed out in another test we could use `-1` or something for the startTime to be able to validate the interval.

Reply:
I added a lock to `PerformanceAndroidEventProcessor` and there are tests to be sent only once, so the async tests would not make much difference now, with the volatile field yes.
there's a new test that checks the value of `getAppStartInterval`

Reply:
`setAppStartEnd` reads from `SystemClock.uptimeMillis()` so it cant be negative